en/docs/learn/enterprise-integration-patterns/messaging-systems/message-router.md

```diff
@@ -1,144 +1,172 @@
+The Message Router performs a logical function (such as an inventory check). It receives a request message (new order), reads it, and routes the request to one of the two recipients according to the content of the message. The router is also defined as a special type of filter.
 
 !!! info
     For more information, see the [Message Router](https://www.enterpriseintegrationpatterns.com/patterns/messaging/MessageRouter.html) documentation.  
 
 ## Sample scenario
 
-The sample scenario depicts an inventory for stocks, and how Message Router EIP routes a message to a different service based on the content of the message. When the router receives a stock request, it reads the content of the request as follows.
+The sample scenario depicts a client sending an appointment reservation request to the Micro Integrator. The message payload of the request contains the name of the hospital where the appointment needs to be confirmed. Based on the hospital name sent in the request message, the Micro Integrator should route the appointment reservation to the relevant hospital's back-end service.
 
-* If the request is made to foo, it is routed to fooOutQueue.
-* If the request is for bar, it is routed to barOutQueue.
+* If the request is made to Grand Oak Community Hospital, it is routed to `GrandOakEP`.
+* If the request is made to Pine Valley Community Hospital, it is routed to `PineValleyEP`.
+* If the request is made to Clemency Medical Center, it is routed to `ClemencyEP`.
 
 !!! note
-    The Switch and Send mediators of the ESB profile of WSO2 EI simulate the Message Router EIP. The Switch Mediator depicts the Router and observes the content of the message, while the Send Mediator sends the message to a selected recipient.
+    The Switch and Call mediators of the WSO2 MI simulate the Message Router EIP. The Switch Mediator depicts the Router and observes the content of the message, while the Call Mediator sends the message to a selected recipient.
 
-    Each case defined in the Switch mediator is used to decide the routing of the message to the appropriate service. fooOutQueue and barOutQueue act as two separate services.
+    Each case defined in the Switch mediator is used to decide the routing of the message to the appropriate service, `GrandOakEP`, `ClemencyEP`, and `PineValleyEP` as three separate services.
 
 ## Synapse configuration
 
-When you unzip the ZIP file you download below in Step 6 when simulating the sample scenario, you can find the below configurations in the `<UNZIPPED_FILE>/src/main/synapse-config` directory. For more information about these artifacts, go to WSO2 EI Documentation.
+When you unzip the ZIP file you download below in Step 4 when simulating the sample scenario, you can find the below configurations in the `<UNZIPPED_FILE>/src/main/wso2mi/artifacts` directory. For more information about these artifacts, go to WSO2 MI Documentation.
```

Comment:
'For more information about these artifacts, go to WSO2 MI Documentation.' Let's point to synapse artifacts page, as we are already in MI docs